### DISCUSSION TITLE
Add-ConditionalFormatting custom color

### DISCUSSION THREAD

**Discussion by RoYoMi** [dfinke/ImportExcel#1276] [Q&A] [answered]
I'd like to use Add-ConditionalFormatting to add a custom RGB color to a field.

I'm confident the command works as long as I select a [system.drawing.color] named value, like this:
`Add-ConditionalFormatting -Worksheet $Worksheet -range $Worksheet.Cells["D1:F5"] -RuleType ContainsText -ConditionValue "Yes" -BackgroundColor LightGreen`

However if I provide the `-BackgroundColor "#e5ebb2"` the fields do not get any color. I also tried a non hex number and this returns the same results. 

Are custom colors even possible?

**Comment by stahler** [comment, marked as answer]
A possible alternative to achieve that color would be:
```powershell
Add-ConditionalFormatting -Worksheet $excel.Sheet1 -Address D:D -DataBarColor ([System.Drawing.Color]::FromArgb(229,235,178))

**Reply by RoYoMi** [reply]
Yep FromArgb method works. I'm not a fan of having to break down the RGB values into 3 distinct integers. But we're in luck because there is an overload that accepts a single int32. So the my desired statement can be rewritten as:

`Add-ConditionalFormatting -Worksheet $WorkSheet -Range $WorkSheet.Cells["D2:$($BottomRight)"] -RuleType ContainsText -ConditionValue 'yes' -BackgroundColor ([System.Drawing.Color]::fromArgb(0xE2EFDA))`

**Reply by geoced** [reply]
You can also use [System.Drawing.ColorTranslator]::FromHtml("#E2EFDA")
